src/data/__mocks__/monteCarlo.js

```diff
@@ -0,0 +1 @@
+export const generateMonteCarloData = jest.fn(() => ({ newCircleTotal: 2, newDartsArray: [[0, 0]], newPiApproximationsArray: [2] }));
```

Comment:
You shouldn't need the jest.fn here since you are using jest.mock on the test file. This may be what is causing the test to fail.